enclave/core/hexdump.c

```diff
@@ -49,16 +49,16 @@ void oe_hex_dump(const void* data, size_t size)
     while (n >= chunk_size)
     {
         oe_hex_string(buf, sizeof(buf), p, chunk_size);
-        OE_TRACE_INFO("%s = ", buf);
+        OE_TRACE(OE_LOG_LEVEL_INFO, "%s\n", buf);
```

Comment:
Can you explain again why this function originally leads to output on stdout even if the callback is enabled?

Reply:
Ok, so the changes here are unrelated to the PR.

The cleanup should just be:
```suggestion
        OE_TRACE_INFO("%s", buf);
```